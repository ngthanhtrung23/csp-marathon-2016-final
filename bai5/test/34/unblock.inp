...1..
..2133
002456
77.456
8999A.
8BBBA.
