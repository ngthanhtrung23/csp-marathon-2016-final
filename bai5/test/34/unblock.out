34
2 U 1
0 R 1
7 R 1
8 U 4
0 L 1
7 L 1
2 D 2
9 L 1
B L 1
4 D 2
1 D 2
3 L 3
1 U 2
4 U 2
5 U 2
6 U 2
A U 2
9 R 3
B R 3
2 D 2
0 R 1
7 R 1
8 D 4
0 L 1
3 L 1
7 L 1
2 U 4
0 R 1
7 R 1
8 U 2
9 L 3
4 D 1
A D 1
0 R 3

